libs/application-generic/src/utils/bridge.ts

```diff
@@ -38,64 +38,68 @@ export const isRegularDigestOutput = (
 
 export const BRIDGE_EXECUTION_ERROR = {
   INVALID_BRIDGE_URL: {
-    code: 'INVALID_BRIDGE_URL',
+    code: 'InvalidBridgeUrl',
```

Comment:
It's easier to read a pascal-cased code on the UI, plus Framework uses PascalCase for all the error codes already and we should be consistent.